### ISSUE TITLE
Close Editor - Stay in frontend

### ISSUE THREAD

**Issue by tinsterman** [ThemeFuse/Brizy#291]
It would be useful to close the editor and stay in the frontend as well as Back to WordPress button

**Comment by michacassola** [comment]
Yes, definitely! It is the first thing that sprung out at me regarding its usability. It is a big pain that you do not have a go back to frontend button. The preview button should have a sister button saying back to frontend next to it.

**Comment by DimiBaitanciuc** [comment]
Back to WP - exists in the on the hamburger menu in the bottom left sidebar. 

Close editor and stay in front end? - sounds interesting but I'm failing to see how it will help. Isn't that basically a preview of the page once the editor closes?

Thanks.

**Comment by michacassola** [comment]
Yes, the back to WP backend button exists in the Hamburger.

One usecase is: You have your site basically finished and want to do a quick edit, maybe some links are missing their URL on a starter site or anything that you notice. You click on "Edit with Brizy" and do your edits and then you just want to go back to the frontend to continue your inspection but what I end up doing now is to erase the javascript from the URL "?edit-brizy" and load the site again. A button next to the preview button would be highly appreciated and I guess not too hard to implement.

Thanks for your consideration!
